azext_iot/central/_help.py

```diff
@@ -44,6 +57,288 @@ def load_central_help():
     _load_central_monitors_help()
     _load_central_command_help()
     _load_central_compute_device_key()
+    _load_central_export_help()
+
+
+def _load_central_export_help():
+    helps[
+        "iot central export"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central data exports.
+    """
+
+    helps[
+        "iot central export list"
+    ] = """
+        type: command
+        short-summary: Get the list of exports for an IoT Central application.
+        examples:
+        - name: List all exports in an application
+          text: >
+            az iot central export list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central export show"
+    ] = """
+        type: command
+        short-summary: Get an export details
+        examples:
+        - name: Get an export details
+          text: >
+            az iot central export show
+            --app-id {appid}
+            --export-id {exportid}
+    """
+
+    helps[
+        "iot central export create"
+    ] = """
+        type: command
+        short-summary: Create an export for an IoT Central application.
+        examples:
+        - name: Create an export from file
+          text: >
+            az iot central export create
+            --app-id {appid}
+            --export-id {exportid}
+            --content './filepath/payload.json'
+
+        - name: Create an export from json payload
+          text: >
+            az iot central export create
+            --app-id {appid}
+            --export-id {exportid}
+            --content "{
+              'displayName': 'Test Export 1',
+              'enabled': false,
+              'source': 'telemetry',
+              'filter': 'SELECT * FROM devices WHERE $displayName != \'abc\' AND $id = \'a\'',
+              'enrichments': {
+                'abc': {
+                  'path': '$templateDisplayName'
+                }
+              },
+              'destinations': [
+                {
+                  'id': 'fa5792a4-ead3-41dc-a972-fbeed33d46ae',
```

Comment:
make this {destinationid} to help tie this to export destination command group

Reply:
Do you means adding a destinationid parameter?

Reply:
I meant as changing the help to be:
`'destinations': [
                {
                  'id': {destination_id},
....`

However, letting the user provide a list of destinations (and breaking up content a bit) would be a good idea too. Similar to how I suggested to break up content in destination - where we would have more arguments that the user can provide and help the user build up the json rather than create the json themselves

Reply:
fixed.